docs/dev/dev_dagster.rst

```diff
@@ -0,0 +1,96 @@
+.. _dev_dagster:
+
+===============================================================================
+Developing with Dagster
```

Comment:
A couple of other things that tripped me up:

* **You can't independently select jobs and asset groups**, meaning if you select a job like `etl_fast_no_epacems`, and then wanted to select an asset group within that job -- say to run just 1 year of data (fast_etl) for all of the assets in the `clean_eia923` group, it doesn't work -- the asset groups are tied to a particular job, and it's one of the Full ETL ones.
* Figuring out how to watch the logs for an asset that was in the midst of processing, or to pull up the logs for a past failed job, took me a while. We should point people at [the Dagit UI docs](https://docs.dagster.io/concepts/dagit/dagit#dagit-ui) at least.

Reply:
I added sections to describe how to view logs and configure asset group executions.

I also created an [issue](https://github.com/dagster-io/dagster/issues/13818) in the dagster repo about filtering asset graphs by asset group names.